docs/product-manuals/zeebe/deployment-guide/getting-started/tutorial-setup.md

```diff
@@ -18,21 +18,21 @@ If this is your first time working with Zeebe, we expect this entire guide to ta
 
 If you're looking for a very fast (but less comprehensive) "first contact" experience, you might prefer the [Quickstart](../local/quickstart.md).
 
-The tutorial assumes you have some basic knowledge of what Zeebe is and what it's used for. If you're completely new to Zeebe, you might find it helpful to read through the ["What is Camunda Cloud?"](/product-manuals/concepts/what-is-camunda-cloud.md) docs article first.
+This tutorial assumes you have some basic knowledge of what Zeebe is and what it's used for. If you're completely new to Zeebe, you might find it helpful to read through [What is Camunda Cloud?](/product-manuals/concepts/what-is-camunda-cloud.md) first.
 
 Below are the components you'll use in the tutorial. The easiest way to run them is to download [Camunda Modeler](https://docs.camunda.io/docs/product-manuals/modeler/camunda-modeler/install-the-modeler) and use the `operate` docker-compose profile in the [zeebe-docker-compose](https://github.com/zeebe-io/zeebe-docker-compose) repository. Further instructions for using Zeebe with Docker can be found in the README.md file in that repository.
 
 You can also download the full distributions for these components, instead of running them with Docker.
 
-1.  [Camunda Modeler](https://docs.camunda.io/docs/product-manuals/modeler/camunda-modeler/install-the-modeler): A desktop modeling tool that we'll use to create and configure our process before we deploy it to Zeebe.
-1.  [Zeebe Distribution](https://github.com/camunda-cloud/zeebe/releases/tag/0.20.0): The Zeebe distribution contains the workflow engine where we'll deploy our process model; the engine is also responsible for managing the state of active process instances. Included in the distro is the Zeebe CLI, which we'll use throughout the tutorial. Please use Zeebe 0.20.0.
+1.  [Camunda Modeler](https://docs.camunda.io/docs/product-manuals/modeler/camunda-modeler/install-the-modeler): A desktop modeling tool we'll use to create and configure our process before we deploy it to Zeebe.
+1.  [Zeebe Distribution](https://github.com/camunda-cloud/zeebe/releases/tag/0.20.0): The Zeebe distribution contains the workflow engine where we'll deploy our process model; the engine is also responsible for managing the state of active process instances. Included in the distro is the Zeebe CLI, which we'll use throughout the tutorial. Use Zeebe 0.20.0.
```

Comment:
**Mandatory:** Wow, we definitely need to change this. They should use the latest version always. Best would be a variable so it always updated with each documentation version. Applies both to the reference in the text, and to the link)

Reply:
Would you prefer to update and link to this: https://camunda.com/blog/2021/02/announcing-zeebe-operate-and-tasklist-1-0-0-alpha1/ ?

Reply:
I would probably rephrase it to "Use the latest release" and then link to this page: https://github.com/camunda-cloud/zeebe/releases
The blog article is already outdated. Right now it would be: https://camunda.com/blog/2021/07/camunda-cloud-110-released/

But in a couple of weeks it will be 1.1.2. 

The link to the github page is a stable one, and then user can pick out the latest release for their needs